packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -203,11 +204,13 @@ export class AwsS3Publish implements PublisherBase {
         async absoluteFilePath => {
           const relativeFilePath = path.relative(directory, absoluteFilePath);
           const fileStream = fs.createReadStream(absoluteFilePath);
+          const fileExtension = path.extname(relativeFilePath);
 
           const params = {
             Bucket: this.bucketName,
             Key: getCloudPathForLocalPath(entity, relativeFilePath),
             Body: fileStream,
+            ContentType: getContentTypeForExtension(fileExtension, false),
```

Comment:
instead of exporting the inner function, we can still use `getHeadersForFileExtension` but get the `Content-Type` prop inside the returned object. let me know if I should do that instead